sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Release History
+
+## 1.0.0-beta.4 (Unreleased)
+### Added
+- Added CommunicationIdentityClient and CommunicationIdentityClientAsync (originally was part of the azure-communication-aministration package).
+- Added support for Azure Active Directory Authentication.
+
+### Breaking
+- CommunicationIdentityTokenScope.Pstn is removed.
```

Comment:
was it `Pstn` or `PSTN`?

Reply:
I've updated this to be more accurate. Java used strings previously and not a enum.